Mathlib/Data/Matroid/IndepAxioms.lean

```diff
@@ -139,88 +142,22 @@ namespace IndepMatroid
 @[simp] theorem matroid_indep_iff {M : IndepMatroid α} {I : Set α} :
     M.matroid.Indep I ↔ M.Indep I := Iff.rfl
 
-/-- An independence predicate satisfying the finite matroid axioms determines a matroid,
-  provided independence is determined by its behaviour on finite sets.
-  This fundamentally needs choice, since it can be used to prove that every vector space
-  has a basis. -/
+/-- If a set `I` is independent iff all its finite subsets are independent, then an `IndepMatroid`
```

Comment:
```suggestion
/-- If a set `I` is independent, i.e. if all its finite subsets are independent, then an `IndepMatroid`
```
assuming that `iff all its finite subsets are independent` is an explanation of what it means for `I` to be independent.